qa/workunits/rbd/import_export.sh

```diff
@@ -136,7 +136,7 @@ if rbd help export | grep -q export-format; then
     rbd import --stripe-count 1000 --stripe-unit 4096 ${TMPDIR}/img testimg
     rbd export --export-format 2 testimg ${TMPDIR}/img_v2
     rbd import --export-format 2 ${TMPDIR}/img_v2 testimg_import
-    rbd info testimg_import|grep "stripe unit"|awk '{print $3}'|grep -Ei '(4K|4096)'
+    rbd info testimg_import|grep "stripe unit"|grep -Ei '(4K|4096)'
```

Comment:
Need “(4K|4 KiB)” for Luminous and “(4K|4 KiB|4096)” for Jewel — or just do the latter and apply to both Luminous and Jewel for simplicity.

Reply:
got it

Reply:
jewel's version seems already not contain the awk bits, more like this: 
`rbd ls -l | grep sparse2 | grep -Ei '(4M|4096k)'` 
so not sure whether we need to bring htis change there?

Reply:
Lumininous and Jewel actually need aa96f44400, c6a747d1d8, and d5b7d672b9. Basically, Jewel will output in the long-form (e.g. 4096), Luminous will output short-hand SI suffix form (e.g. 4K), and Mimic will output in IEC suffix form (e.g. 4 KiB).